src/backend/nodes/cypher_readfuncs.c

```diff
@@ -294,7 +294,7 @@ void read_cypher_merge_information(struct ExtensibleNode *node)
 {
     READ_LOCALS(cypher_merge_information);
 
-    READ_UINT_FIELD(flags);
+    READ_INT_FIELD(flags);
```

Comment:
Why were these changed back to INT from UINT. They were purposely changed to UINT in the master.